README.md

```diff
@@ -150,6 +150,9 @@ OPTIONS:
         --bin <NAME>                   Binary to run
         --example <NAME>               Example binary to run
         --features <CARGO-FEATURES>    Features to pass to cargo
+        --manifest-path <PATH>         Path to Cargo.toml
```

Comment:
This looks like it exactly matches the `--manifest-path` help documentation from `cargo run -h`